rollup.yaml

```diff
@@ -221,6 +221,35 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
+  /gio:
+    post:
+      operationId: gio
+      summary: General Input/Output
```

Comment:
I think it is "generic" instead of "general". Can you check with Milton. Now that I think of it, I don't know if we have been consistent throughout.